source/userguide/active_learning.rst

```diff
@@ -0,0 +1,226 @@
+Active Learning in MindMeld
+===========================
+
+Building new conversational applications, adding new domains and intents for existing applications or improving performance based on user experience, all of these benefit from the addition of more informative data to the trained models. Data generation resources and especially user logs for the application are highly beneficial in capturing new trends and growing diversity in terms of the phrasing of queries. While adding both old and new data to the trained models can improve system performance in terms of accuracy, it reduces scalability by increasing the computation and time costs.
+
+Active Learning is an approach to continuously and iteratively learn the best data for training the models while preserving good accuracies and maintaining best-case tradeoffs with respect to computational power and time. MindMeld provides an inbuilt functionality to select the must-have queries from existing data or additional logs and datasets to get the best out of your conversational applications.
```

Comment:
```suggestion
Active Learning is an approach to continuously and iteratively pick data that is most informative for the models to train on from a pool of unlabelled data-points. By picking such data-points, active learning provides a higher chance of improving the accuracy of the model with fewer training examples. It also greatly reduces the number of queries to be annotated by humans. MindMeld provides this inbuilt functionality to select these must-have queries from existing data or additional logs and datasets to get the best out of your conversational applications.
```

Reply:
"while preserving good accuracies and maintaining best-case tradeoffs with respect to computational power and time" - don't think this is something we can guarantee.

Reply:
Makes sense, higher chance sounds reasonable!